From: Federico Ceratto <federico@debian.org>
Subject: support multiarch and disable RPATH
Forwarded: no

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,6 +14,8 @@
 #project(SRT VERSION "1.3.2")
 project(SRT C CXX)
 
+include(GNUInstallDirs)
+
 set (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts")
 include(haiUtil)
 include(FindPkgConfig)
@@ -681,6 +683,8 @@
 	# We state that Darwin always uses CLANG compiler, which honors this flag the same way.
 	set_target_properties(${name} PROPERTIES COMPILE_FLAGS "${CFLAGS_CXX_STANDARD} ${EXTRA_stransmit}" ${FORCE_RPATH})
 
+set_target_properties(${name} PROPERTIES SKIP_BUILD_RPATH TRUE)
+
 	target_link_libraries(${name} ${srt_link_library})
 endmacro()
 
