From: Federico Ceratto <federico@debian.org>
Date: Fri, 13 Sep 2019 12:45:29 +0200
Subject: support multiarch and disable RPATH

Forwarded: no
---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ec91d08..5302a5c 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,6 +14,8 @@ cmake_minimum_required (VERSION 2.8.12 FATAL_ERROR)
 #project(SRT VERSION "1.3.4")
 project(SRT C CXX)
 
+include(GNUInstallDirs)
+
 set (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts")
 include(haiUtil)
 include(FindPkgConfig)
@@ -757,6 +759,8 @@ macro(srt_make_application name)
 	# We state that Darwin always uses CLANG compiler, which honors this flag the same way.
 	set_target_properties(${name} PROPERTIES COMPILE_FLAGS "${CFLAGS_CXX_STANDARD} ${EXTRA_stransmit}" ${FORCE_RPATH})
 
+set_target_properties(${name} PROPERTIES SKIP_BUILD_RPATH TRUE)
+
 	target_link_libraries(${name} ${srt_link_library})
 	if (USE_GNUSTL)
 		target_link_libraries(${name} PRIVATE ${GNUSTL_LIBRARIES} ${GNUSTL_LDFLAGS})
